apps/agora/app/server.ts

```diff
@@ -70,22 +62,15 @@ export function app(): express.Express {
 }
 
 function run(): void {
+  const port = process.env['PORT'] || '4200';
+
   // Start up the Node server
   const server = app();
-  server.listen(PORT, () => {
-    console.log(`Node Express server listening on http://localhost:${PORT}`);
+  server.listen(port, () => {
+    console.log(`Node Express server listening on http://localhost:${port}`);
   });
 }
 
-// Webpack will replace 'require' with '__webpack_require__'
-// '__non_webpack_require__' is a proxy to Node 'require'
-// The below code is to ensure that the server is run only when not requiring the bundle.
-/* eslint-disable camelcase,no-undef */
-declare const __non_webpack_require__: NodeRequire;
-const mainModule = __non_webpack_require__.main;
-const moduleFilename = (mainModule && mainModule.filename) || '';
-if (moduleFilename === __filename || moduleFilename.includes('iisnode')) {
```

Comment:
Removed the following as we won't be needing it soon:

See: https://medium.com/@pyaesone/what-is-require-main-module-in-node-js-means-61b0ddca1a0d